src/api/api.js

```diff
@@ -2,27 +2,33 @@ import axios from 'axios';
 
 // const baseURL = 'http://198.199.100.209:5005/api';
 const baseURL = 'http://localhost:5005/api/v1.0';
-const getRequestConfig = () => ({
-  headers: {
-    'Content-Type': 'application/json',
-    Authorization: 'token 10cbb8be976d8db1efcca8c60fd7e3594f74772fd877d36afc825ed6df3b7829',
-    // Need to implement login register
-    // 'Authorization': store.getState().auth.token
-  },
-  data: {},
-});
+const getRequestConfig = () => {
+   return {
+      headers: {
+         'Content-Type': 'application/json',
+         'Authorization': 'token 10cbb8be976d8db1efcca8c60fd7e3594f74772fd877d36afc825ed6df3b7829'
+         // Need to implement login register
+         // 'Authorization': store.getState().auth.token
+      },
+      data: {}
+   }
+};
 
 const urls = {
-  charts: `${baseURL}/charts`,
-  transactions: `${baseURL}/transactions`,
+   charts: baseURL + '/charts',
+   transactions: baseURL + '/transactions',
+   portfolios: baseURL + '/portfolios',
+   leagues: baseURL + '/leagues'
```

Comment:
Why did you change it not to have the `${baseURL}` format?

Reply:
not sure, will fix